vscode/package.json

```diff
@@ -31,13 +31,13 @@
     "walkthroughs": [
       {
         "id": "qsharp-vscode.welcome",
-        "title": "The Azure Quantum Development Kit",
-        "description": "Getting started with the Azure Quantum Development Kit in VS Code",
+        "title": "The Microsoft Quantum Development Kit",
```

Comment:
Maybe just "Microsoft Quantum Development Kit"? (or are we going for something like "The Batman" 😜)